src/search/containers/search-results/search-results.component.ts

```diff
@@ -3,7 +3,6 @@ import { ActivatedRoute, Router } from '@angular/router';
 import { Jurisdiction } from '@hmcts/ccd-case-ui-toolkit';
 import { combineLatest, Subscription } from 'rxjs';
 import { JurisdictionService } from '../../../app/services/jurisdiction/jurisdiction.service';
-import { SearchStatePersistenceKey } from '../../enums';
 import { SearchResult, SearchResultDisplay } from '../../models';
 import { SearchService } from '../../services/search.service';
 import { NoResultsMessageId } from '../../enums';
```

Comment:
@johnbenjamin-hmcts - I think the import ordering might need checking (would expect `../../enums` to come before `../../models`).